src/main/java/com/bootcamp/demo/business/AccountService.java

```diff
@@ -30,45 +30,87 @@ public void setEncryptor(Encryptor encryptor) {
         this.encryptor = encryptor;
     }
 
-    public boolean accountValidation(Account account){
-        if(account.getName().isEmpty() || account.getPassword().isEmpty() || account.getEmail().isEmpty() || account.getUsername().isEmpty() || account.getAddress().isEmpty() || account.getPhoneNumber().isEmpty() || account.getDateOfBirth().isEmpty())
-            return false;
+    public ValidationResponse accountValidation(Account account){
 
-        String expression = "^([A-Z][a-z]*((\\s)))+[A-Z][a-z]*$";
-        if(!account.getName().matches(expression))
-            return false;
+        ValidationResponse validationResponse = new ValidationResponse();
 
-        expression = "[a-zA-Z0-9]*$";
-        if(!account.getUsername().matches(expression))
-            return false;
+        if(account.getName().isEmpty())
```

Comment:
if one of the fields is null this type of validation will result in NPE. Might avoid this using `StringUtils.isEmpty(value)`. This type of issues might be easily spotted with unit tests